shell/app/config-page/components/text/text.tsx

```diff
@@ -84,12 +85,12 @@ const Text = (props: CP_TEXT.Props) => {
                     className={`${textClassNames} hover-active`}
                   >
                     {tText}
-                    {icon && <CustomIcon className={`mr4 ml4 ${iconStyleName}`} type={icon} />}
-                    {image && <img src={image.startsWith('/images') ? imgMap[image] : image} className='text-image' />}
+                    {icon && <ErdaIcon type="Icon" props={{ iconType: icon, className: `mr4 ml4 ${iconStyleName}` }} />}
```

Comment:
the usage Of `ErdaIcon` is so strange... 
@michael8512  seems `type` is not using, why do we need pass a nested props？

Reply:
It is a component of config-page

Reply:
> It is a component of config-page

so what? I didn't find out usage of `type="Icon"`

![image](https://user-images.githubusercontent.com/5175455/117596906-00592980-b177-11eb-9845-c255102b3514.png)

Reply:
type can be removed

Reply:
> type can be removed

My key point is why do we need to pass a nested props. what's the difference if flatten the props？